src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -585,6 +586,28 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP105",
                 $"Unable to load file from URI \"{fileUri}\".");
+                
+            public ErrorDiagnostic FunctionDoesNotExistInNamespace(Symbol namespaceSymbol, string name) => new ErrorDiagnostic(
+                TextSpan,
+                "BCP106",

```

Comment:
This is the same as BCP086 was previously.

Reply:
Not an issue - just wondering why

Reply:
I think I just wanted the with/without suggestion errors to be next to each other. Honestly, although it makes testing uniqueness a little harder, I wonder if we should be using the same code for with/without suggestions, so that external information on different diagnostic codes doesn't get fragmented.

Reply:
Yeah that's a good point. I've been wondering that myself. One way to validate it would be that all messages with the same code have the same prefix text. (Shortest message in the group is the prefix - all others must start with it.)